meta/saiserialize.cpp

```diff
@@ -1636,6 +1636,14 @@ std::string sai_serialize_object_meta_key(
     return key;
 }
 
+std::string sai_serialize_switch_attr(
```

Comment:
please remove we already have generic function to serialize attributes for all objects